src/main.rs

```diff
@@ -371,15 +372,31 @@ impl FileReceiver {
     }
 }
 
-fn main_recv(addr: &str, n_conn: &str) -> Result<()> {
+fn main_recv(
+    addr: &str,
+    n_conn: &str,
+    ask_for_confirmation: bool,
+    protocol_version: u16,
```

Comment:
This (and main_send) start getting a bit cumbersome, maybe they should receive a struct?

Reply:
:woman_shrugging: It looks fine to me, I don’t have a strong opinion on it. We could remove `protocol_version` entirely since it’s a constant anyway, but I suppose it’s there for testing purposes?

Reply:
yeah, it's only here for testing.